File: BinaryTree/java/treeIncludes.java
import java.util.ArrayDeque;
import java.util.Queue;

class TreeIncludes {
  public static boolean treeIncludes(TreeNode root, int target) {
    if (root == null) {
      return false;
    }

    Queue<TreeNode> queue = new ArrayDeque<>();
    queue.add(root);
    while (!queue.isEmpty()) {
      TreeNode current = queue.remove();
      if (current.val == target) {
        return true;
      }
      if (current.left != null) queue.add(current.left);
      if (current.right != null) queue.add(current.right);
    }

    return false;
  }

  public static void run() {
    // intentionally empty
  }
}

File: BinaryTree/php/treeIncludes.php
<?php

declare(strict_types=1);

require_once __DIR__ . '/TreeNode.php';

function treeIncludes(?TreeNode $root, int $target): bool
{
    if ($root === null) {
        return false;
    }

    $queue = [$root];
    while ($queue) {
        /** @var TreeNode $current */
        $current = array_shift($queue);
        if ($current->val === $target) {
            return true;
        }
        if ($current->left !== null) {
            $queue[] = $current->left;
        }
        if ($current->right !== null) {
            $queue[] = $current->right;
        }
    }

    return false;
}
